src/setup.c

```diff
@@ -64,13 +64,17 @@ static void setup_clock(void)
  *
  * Exception priorities:
```

Comment:
Great! update also the table on doc about exceptions

Reply:
vaya morro ;P

Reply:
https://github.com/Bulebots/bulebule/issues/369